Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.9-slim AS build-image
+
+WORKDIR /app/
+
+# copy requirements file (should include selected versions of uvicorn gunicorn)
+COPY ./requirements.txt /app/requirements.txt
+
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential \
+    libcairo2 \
+    && pip install --no-cache-dir --upgrade pip setuptools wheel \
+    && pip install --no-cache-dir --user -r /app/requirements.txt
+
+FROM python:3.9-slim AS runtime-image
+LABEL maintainer="RCSB IT <it@rcsb.org>"
+
+WORKDIR /app/
```

Comment:
I think this can be removed since it's already set up above

Reply:
Since we're using a multi-stage build process for the docker images, after the second `FROM` command in line 13 all the other commands afterwards are basically running in a new environment. So setting the `WORKDIR` again is needed.

Reply:
Ah OK, my bad. Thanks for explaining